resources/lib/mvupdate.py

```diff
@@ -391,20 +391,25 @@ class UpdateApp(AppLogger):
     """ The standalone updater application class """
 
     def __init__(self):
+        scriptpath, scriptname = os.path.split(sys.argv[0])
+        storederr = None
+        version = '0.0'
         try:
-            self.mypath = os.path.dirname(sys.argv[0])
-            tree = ET.parse(self.mypath + '/addon.xml')
+            tree = ET.parse(os.path.join(scriptpath, 'addon.xml'))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using xml.etree.ElementTree.parse to parse untrusted XML data is known to be vulnerable to XML attacks. Replace xml.etree.ElementTree.parse with its defusedxml equivalent function or make sure defusedxml.defuse_stdlib() is called](https://app.codacy.com/manual/MediathekView/plugin.video.mediathekview/pullRequest?prid=5459637)

Reply:
Wäre mir das mal vor #155 aufgefallen. :(

Reply:
Das ist eine lokale Resource aus dem Filesystem...da wollen wir mal die Kirche im Dorf lassen :-)